### DISCUSSION TITLE
Fewer visits and page views than Plausible Analytics

### DISCUSSION THREAD

**Discussion by vitobotta** [umami-software/umami#1764] [General]
I am self hosting Umami now (have changed name of the script and endpoint so uBlock origin etc don't block it) and comparing the data with Plausible Analytics (hosted version) Umami reports fewer visits and page views, a 5-10% difference. What can explain this? Thanks

**Comment by vitobotta** [comment]
I wonder if it's because of the bots detection. How does this work in Umami? @mikecao

**Reply by briancao** [reply]
Bot check is the first thing that occurs during data collection.  We're using the isbot package to determine if the visitor is a bot or not.  A rough 5% difference appears to be the norm when other users have compared between Plausible and Umami.  You can use an environment variable 'DISABLE_BOT_CHECK=true' if you want to have all visitors tracked.

[https://github.com/umami-software/umami/blob/master/pages/api/collect.js (line ](https://github.com/umami-software/umami/blob/7a3443cd06772f3cde37bdbb0bf38eabf4515561/pages/api/collect.js#L13)

**Comment by Fawesum** [comment]
Umami also reports 20% fewer unique visitors than Google Analytics for me (but more pageviews). A year ago it reported 20% more than GA.

**Reply by briancao** [reply]
Without knowing what GA is doing under the hood, it doesn't serve as an appropriate baseline for which tracker between GA or Umami is more accurate.  For Umami, you can see the 'collect' in the network tab, count each visit as it records into your database, and potentially follow the code for each visit.

Maybe after v2 release, we'll have some time to dedicate in researching the difference in collection rates between Umami and other platforms.  For now, it's very low priority.
